From 2a8dbd3840c7b01e7c544035749d3fde893923ed Mon Sep 17 00:00:00 2001
From: "Richard W.M. Jones" <rjones@redhat.com>
Date: Sat, 12 Jul 2025 18:12:42 +0100
Subject: [PATCH] todo: Remove a couple of minor features that have been
 implemented

Rust was implemented in 2023.

nbdcopy implemented page cache efficient operations.

(cherry picked from commit fe284d59fa0e5a85a4abac418efb8b79d81cdbb5)
---
 TODO | 6 ------
 1 file changed, 6 deletions(-)

diff --git a/TODO b/TODO
index e140b4fd..426b0384 100644
--- a/TODO
+++ b/TODO
@@ -1,10 +1,6 @@
 Explore if nbd_aio_notify_error is needed for faster response if
 server goes away.
 
-Bindings in other languages.
- - Latest attempt at adding Rust:
-   https://www.redhat.com/archives/libguestfs/2019-August/msg00416.html
-
 Example code integrating with ppoll, pollfd, APR pollset (and others?).
 
 NBD resize extension.
@@ -32,8 +28,6 @@ nbdcopy:
  - Synchronous loop should be adjusted to take into account
    the NBD preferred block size, as was done for multi-thread loop.
  - Benchmark.
- - Better page cache usage, see nbdkit-file-plugin options
-   fadvise=sequential cache=none.
  - Consider io_uring if there are performance bottlenecks.
  - Configurable retries in response to read or write failures.
 
-- 
2.47.3

